pytest_django/asserts.py

```diff
@@ -213,6 +230,16 @@ def assertNumQueries(
     ):
         ...
 
+    if USE_CONTRIB_MESSAGES:
+
+        def assertMessages(
+            response: HttpResponseBase,
+            expected_messages: Sequence[Message],
+            *args,
+            ordered: bool = ...,
+        ) -> None:
+            ...
```

Comment:
This condition doesn't have an affect inside of TYPE_CHECKING, so let's remove it.

```suggestion
    # Added in Django 5.0.
    def assertMessages(
        response: HttpResponseBase,
        expected_messages: Sequence[Message],
        *args,
        ordered: bool = ...,
    ) -> None:
        ...
```